contrib/test/crio-integration-playbook.yaml

```diff
@@ -1,12 +1,67 @@
 ## This playbook expects --extra-vars "commit=<commit>"
 ## and either --extra-vars "pullrequest=<PR #>" or
-## --skip-tags pr
+## --skip-tags pr.
+##
+## Required: --extra-vars extras_repo_baseurl=<HTTP or HTTPS>
```

Comment:
Uggg, that should also include mention of ``server_repo_baseurl`` being required.  Not fixing it in the interest of getting a run going quickly...